process/labs-process.md

```diff
@@ -0,0 +1,121 @@
+
+Archived lab repositories are not actively maintained and will be marked as
+"archived" (read-only) on GitHub. They can be reactivated if there is interest
+in resuming work on a project.
+
+## New Lab Proposal Process
+
+1. Fork the `<TBD>` repo.
+
+2. Fill out the [proposal template](templates/LAB_NAME_lab_stage.md)
+   and save it into the `labs` subdirectory under the name of your lab,
+   such as `coolnewproject_lab.md`.
+   <br/>
+> [!TIP]
+> It is expected that your lab repository on GitHub will have the same
+> name as the proposal, so keep that in mind when submitting your proposal.
+
+3. In the proposal template, there is an entry for sponsor(s). Although this
+   is not required, proposers are encouraged to seek a sponsor in the OpenSSF
+   community who can help them create ties with the rest of the community
+   and review the proposal to make sure it is novel and aligned with the
+   [OpenSSF mission].
+   <br/>
+   To find sponsors:
+     1. use your connections to existing projects and ask maintainers,
+     2. engage with [existing Technical Initiatives] (working groups, projects
+	    or SIGs) with affinities to the proposed lab and pitch the project in
+		their meetings or [Slack channels](https://slack.openssf.org/). It's
+		good to have the template already filled out when you reach out.
+   <br>
+> [!IMPORTANT]
+> Lab sponsors may but are not required to actively participate in
+> the lab once the proposal has been reviewed and accepted.
+
+4. Commit your changes with proper sign-off. This means that your commit
+   log message must contain a line that looks like the following one,
+   with your actual name and email address:
+
+   `Signed-off-by: John Doe <john.doe@example.com>`
+
+   Adding the `-s` flag to your `git commit` command will add that line
+   automatically. You can also add it manually as part of your commit
+   log message or add it afterwards with `git commit --amend -s`.
+
+5. Submit a Pull Request to the `<TBD>` repo.
+
+The [OpenSSF TAC] will then review your proposal. Like sponsors, TAC members may
+but are not required to participate in ongoing work like contributing or
+reviewing code in the lab.
+
+### Transferring an existing repository
+
+By default, OpenSSF staff will create a new GitHub repository for you to
+start a new lab in. If you have an existing GitHub repo you would like to
+bring to your proposed lab, you have the option to request for that
+repo to be transferred into the <TBD GH org> instead.
+
+However, we require that every commit in the existing repo to br
+signed-off so there are no issues related to [DCO].
+If that is not the case, you will need to transfer your existing code by
+squashing all of your commits into a single first commit made against
+your new lab repo with your sign-off.
+
+### License requirement
+
+<!-- This is something that other Labs require of the projects they host. What, if any requirements should we include here? -->
```

Comment:
I suggest that we apply the very same IP requiremnents as defined in the OpenSSF Charter Section 4: 

i) Software source code
(1) Apache License, Version 2.0, available at [https://www.apache.org/licenses/LICENSE- 2.0](https://www.apache.org/licenses/LICENSE-%202.0); or
(2) MIT License available at https://opensource.org/licenses/MIT;
ii) Data
(1) Any of the Community Data License Agreements, available at https://www.cdla.io/;
iii) Specifications
(1) Community Specification License, Version 1.0, available at https://github.com/CommunitySpecification/1.0
iv) All other Documentation
(1) Creative Commons Attribution 4.0 International License, available at https://creative commons.org/licenses/by/4.0/.

Reply:
done.